app/src/lib/stores/app-store.ts

```diff
@@ -1514,15 +1369,11 @@ export class AppStore extends TypedBaseStore<IAppState> {
       return defaultPRLabel
     }
 
-    const repositoryState = this.repositoryState.get(repository.hash)
-
-    if (!repositoryState) {
```

Comment:
Extracting out the repository state cache showed this was an inconsistent way of poking at the repository state cache, and it would bypass the behaviour in `this.getRepositoryState()` that creates the initial state if it didn't already exist.